.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,14 +31,18 @@ If using docker:
 **Problem description**:
 
 <!-- 
-Please be precise about errors. Instead of saying "doesn't work" or "fails",
-describe the behavior you're observing and what you'd expect to see.
+Please don't say "doesn't work" but instead write a precise description of the
+behavior you observe and explain what behavior you'd expect.
 
-Include relevant logs and error messages or stacktraces if possible.
+If a query returns an error message, please include the error message and a
+stacktrace of the error.  To get the stack trace either run the query in crash
+with \verbose on, or run it in the AdminUI Console with `[x] Show error trace`
+set.
 
-If there are memory issues please provide the garbage collection log file if
-possible.
+If a query doesn't return an error but behaves in an unexpected way, please
+include the server logs.
 
+If there are performance issues, please provide the garbage collection logs.
```

Comment:
would change this to active voice, e.g., "If you are experiencing performance issues, ..."